diag_test.py

```
@@ -790,18 +960,19 @@ def interpret_icc(v):
-                            Types: 'table', 'plot', 'text', 'contingency_table', 'html'
-    
-    Returns:
-        str: HTML report string
+    Styled to match Shiny application theme.
     """
     html_parts = []
     
-    # Header
+    # ดึงสีจาก Palette กลาง เพื่อให้ตรงกับส่วนอื่นของเว็บ
+    primary_color = COLORS.get('primary', '#0056b3')
+    primary_light = '#e3f2fd' # สีพื้นหลังหัวตารางที่อ่อนลง ให้ดู modern ขึ้น
```

Ruff (0.14.10)

Local variable `primary_light` is assigned to but never used

Remove assignment to unused variable `primary_light`

(F841)